v3/setup/images-control.sh

```diff
@@ -6,11 +6,11 @@ if [ "${NODE_ROLE}" != "control" ]; then
     exit 0
 fi
 
-if [ "$(etcdctl get images-control-bootstrapped)" == "true" ]; then
+if [ "$(etcdctl get /bootstrap.service/images-control-bootstrapped)" == "true" ]; then
```

Comment:
hmmm not totally sure. I don't see where we would use `bootstrap.service/images/control` for though.